deploy/frontend.yaml

```
@@ -19,36 +21,29 @@ objects:
         paths:
           - /apps/subscriptions-dashboard-ui
       image: ${IMAGE}:${IMAGE_TAG}
-      navItems:
-        - appId: 'subscriptionsDashboardUi'
-          title: 'Overview'
-        - appId: 'subscriptionsInventory'
-          title: 'Subscriptions Inventory'
-        - appId: 'subscriptionsUsage'
-          title: 'Subscriptions Usage'
-          expandable: true
-          permissions:
-            - method: 'isEntitled'
-          routes:
-            - appId: 'rhel'
-              title: 'RHEL'
-              href: '/insights/subscriptions/rhel'
-        - appId: hybridCommittedSpend
-          title: 'Hybrid Committed Spend'
-          expandable: true
-          routes:
-            - appId: 'hybridCommittedSpend'
-              title: 'Hybrid Committed Spend'
-              href: '/business-services/hybrid-committed-spend'
-
       module:
         manifestLocation: '/apps/subscriptions-dashboard-ui/fed-mods.json'
+        defaultDocumentTitle: Subscription Dashboard | Subscription Dashboard
         modules:
-          - id: 'subscriptionsDashboardUi'
+          - id: subscriptions-dashboard-ui
             module: './RootApp'
             routes:
               - pathname: /subscriptions
               - pathname: /subscriptions/overview
+        bundleSegments:
+        - segmentId: subscriptions-dashboard-ui
+          bundleId: subscriptions
+          navItems:
+            - title: Overview
+              id: overview
+              href: "/subscriptions/overview"
+              icon: SubscriptionsIcon
+          position: 300
+
+        searchEntries:
```

This needs one less indentation
```suggestion
      searchEntries:
```